src/devices/curtains.ts

```diff
@@ -315,8 +312,8 @@ export class Curtain {
     if (this.platform.config.credentials?.openToken) {
       this.debugLog(`Curtain: ${this.accessory.displayName} OpenAPI parseStatus`);
       // CurrentPosition
-      this.setMinMax();
       this.CurrentPosition = 100 - Number(this.slidePosition);
+      this.setMinMax();
```

Comment:
Thoughts on if `if (!this.setNewTarget) {` should be removed on line 336?

Reply:
It looks to be redundant, so I would say yes 👍.

Reply:
that's what I figured, nice catch. I have pushed a new beta. Can you test this out and report on #123?